src/curator/toolforge.py

```
@@ -2,7 +2,7 @@
 import os
 from typing import Dict, Any, Callable, TypeVar, Optional
 
-import requests
+import httpx
```

Pylint (3.3.7)

[error] Unable to import 'httpx'

(E0401)